Vagrantfile

```diff
@@ -0,0 +1,23 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "CentOS64"
+  config.vm.box_url = "http://puppet-vagrant-boxes.puppetlabs.com/centos-64-x64-vbox4210.box"
+
+  config.vm.network "forwarded_port", guest: 80, host: 8080
+
+  config.vm.provider "virtualbox" do |vb|
+    vb.memory = "1024"
+  end
+
+  config.vm.provision "ansible" do |ansible|
+      ansible.playbook = "ansible/cfgov_refresh_setup.yml"
+      ansible.verbose = 'v'
+      ansible.groups = {
+          "db" => ["default"],
```

Comment:
Vagrant configuration gives the 1 local box all the roles of the major components of the entire app stack.